SCClassLibrary/Common/Audio/InOut.sc

```diff
@@ -237,10 +249,12 @@ AbstractOut : UGen {
 Out : AbstractOut {
 	*ar { arg bus, channelsArray;
 		channelsArray = this.replaceZeroesWithSilence(channelsArray.asUGenInput(this).asArray);
+		this.checkChannelCount(bus, channelsArray);
```

Comment:
this will never be called, because the line before:
`this.replaceZeroesWithSilence(channelsArray.asUGenInput(this).asArray);`
already has converted the bus to its index.